package.json

```diff
@@ -93,7 +93,6 @@
     "beautify-lint": "beautify-lint 'lib/**/*.js' 'hot/**/*.js' 'bin/**/*.js' 'benchmark/*.js' 'test/*.js'",
     "nsp": "nsp check --output summary",
     "cover": "node --harmony ./node_modules/.bin/istanbul cover -x '**/*.runtime.js' node_modules/mocha/bin/_mocha",
-    "beautify": "beautify-rewrite 'lib/**/*.js' 'hot/**/*.js' 'bin/**/*.js' 'benchmark/*.js' 'test/*.js'",
```

Comment:
I'm not sure, I think I meant to replace it with `npm run lint -- --fix`, but looks like I forgot that.